test/e2e/bpmn.rendering.test.ts

```diff
@@ -129,3 +130,19 @@ describe('no visual regression', () => {
     expect(image).toMatchImageSnapshot(imageSnapshotConfigurator.getConfig(fileName));
   });
 });
+
+it.each([
+  [FitType[FitType.Default], FitType.Default],
+  [FitType[FitType.Horizontal], FitType.Horizontal],
+  [FitType[FitType.Vertical], FitType.Vertical],
+  [FitType[FitType.HorizontalVertical], FitType.HorizontalVertical],
+])(`load options: fit %s`, async (title: string, fitType: FitType) => {
+  const bpmnDiagramPreparation = new BpmnDiagramPreparation(new Map<string, BpmnLoadMethod>([]), { name: 'non-regression' }, 'non-regression', { fitType });
+  const pageTester = new PageTester(bpmnDiagramPreparation, 'viewport', 'BPMN Visualization Non Regression');
+
+  const fileName = 'tasks';
```

Comment:
:warning: we should use a dedicated tests here. We don't want changes in the 'tasks' rendering impact this test.
I suggest to create dedicated diagrams here: at least one with very different values for horizontal and vertical dimensions to highlight fit options.

Reply:
ok